middleware/service/unittest/source/manager/test_manager.cpp

```diff
@@ -1226,5 +1226,43 @@ namespace casual
          }
       }
 
+      TEST( service_manager, runtime_update_routes)
+      {
+         common::unittest::Trace trace;
+
+         auto domain = local::domain( R"(
+domain:
+   services:
+      -  name: a
+         routes: [ b]
+)");
+
+         {
+            auto state = local::call::state();
```

Comment:
Hmm, SM should have a `state()` function in `unittest/utillity.h` that we should use?

Reply:
Hence, remove the `local::call::state()` function -> alter the stuff in this file that uses it.